cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/NativeProvisioner.java

```diff
@@ -35,7 +36,7 @@
 public class NativeProvisioner implements Provisioner {
 
   public static final ProvisionerSpecification SPEC = new ProvisionerSpecification(
-    "native", "Native",
+    Profile.NATIVE_NAME, "Native",
```

Comment:
The constant should live in NativeProvisioner, not in Profile.

Reply:
But Profile cannot access NativeProvisioner, we cannot use that Constant in Profile so it will be no use.

Reply:
ok I see.